dockers/docker-base/Dockerfile.j2

```diff
@@ -25,12 +25,14 @@ RUN apt-get clean && apt-get update
 RUN apt-get -y install \
     rsyslog            \
     supervisor         \
-    vim-tiny           \
+    vim                \
```

Comment:
The largest component for vim is vim-runtime 25.5M: https://packages.debian.org/jessie/vim-runtime. The total size for vim packages is ~28M.